### DISCUSSION TITLE
Unfortunately, we were unable to verify your academic affiliation based on the information you submitted

### DISCUSSION THREAD

**Discussion by Abdulmhasan** [community/community#52155] [GitHub Education]
hi i keep getting this message please help my student Id in Arabic But it Has QR code In English How do I contact Admins

**Comment by Abdulmhasan** [comment]
?

**Comment by samus-aran** [comment]
Hi @Abdulmhasan 

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

Here is a quick tip on submitting evidence:

- If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
- Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

We don't currently don't process QRs so you will need to provide additional info to proceed.

Let us know how you get on 👍

**Comment by Abdulmhasan** [comment]
Sent from Mail<https://go.microsoft.com/fwlink/?LinkId=550986> for Windows

Hi this is my info in English with id
I cant do it by myself because there is no upload image in the GitHub
***@***.***


Hi @Abdulmhasan<https://github.com/Abdulmhasan>

We'd love to help! You are more likely to get a useful response if you are explicit about the feedback given to you from the rejected application submission. Do you have a screenshot of the rejection reasons? This might help someone give you a nudge in the right direction.

Here is a quick tip on submitting evidence:

  *   If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
  *   Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

We don't currently don't process QRs so you will need to provide additional info to proceed.

Let us know how you get on 👍
